package.json

```diff
@@ -21,23 +21,31 @@
     "elasticio-component"
   ],
   "dependencies": {
-    "elasticio-node": "0.0.5",
-    "elasticio-sailor-nodejs": "2.1.0",
-    "lodash": "^4.6.1",
+    "dotenv": "6.2.0",
+    "elasticio-node": "0.0.9",
+    "elasticio-sailor-nodejs": "^2.3.0",
```

Comment:
Should we use here `sailor 2.2.3` with flow-control feature?

Reply:
New sailor didn't help to resolve the problem with OOM, so I made the decision to leave current sailor